### DISCUSSION TITLE
[Scam Alert] NOKON نوکون

### DISCUSSION THREAD

**Discussion by Webamoozcom** [Webamoozcom/warning-list#17] [Announcements]
<div dir="rtl">

**Project Name:** 
NOKON

**نام پروژه:** 
نوکون

**نماد پروژه:**
Nko

**اتفاقی که افتاده است:**
ico بدون وایت پیپر و هیچ مبنایی

**قوانینی که نقض شده است:**

**نوع هشدار:**
هشدار کلاهبرداری

**نوع کلاهبرداری:** 
ico دروغین

**وضعیت کلاهبرداری:**
در جریان

**نام کلاهبرداران:**

**نام شرکت(ها):**

**آدرس وب سایت(ها):**
https://nokon.org/
https://t.me/crypto_news_fa

**برآورد مبلغ کلاهبرداری:**

**منابع:**
https://t.me/webamoozir/12599

**جمع بندی:**

**Comment by Webamoozcom** [comment]
https://www.blockchain.com/eth/address/0x259562c54c07aca61e12ee12c62016eaf3fd7852
تا الان حدود 156 اتریوم جمع آوری کرده،
جالبه هیچ اتری را هنوز جابجا نکرده و 
فقط داره جمع میکنه...

**Comment by Webamoozcom** [comment]
کانال تلگرام اصلی جهت تبلیغات این توکن تقلبی بقرار زیر است:
crypto_news_fa
یک گروه تلگرامی با بیش از 93 هزار نفر عضو، که از تاریخ 10 آگوست به بعد در قالب فروش توکن جعلی "نوکون" در اختیار کلاهبرداران قرار گرفته یا از ادمین قبلی خریداری شده است. از اون تاریخ به بعد، روزی حداقل 7 بار انتهای هر پست تبلیغ توکن جعلی را میکنه!!!

**Comment by Webamoozcom** [comment]
ارزهای دیجیتال شما را ثروتمند میکند🚀💰🌪

🌪پیش فروش ارز دیجیتال نوکون(nokon) شروع شده است🌪😈

💣خرید در پیش فروش، بهترین فرصت برای خرید ارز دیجیتال نوکون میباشد و بعد از پیش فروش رشد و سود چند هزار درصدی را تجربه میکنید(پیش بینی سود هزاران درصدی)💣

آموزش خرید👇h
http://t.me/crypto_news_fa/2775

مرجع اخبار ارزهای دیجیتال،کریپتو نیوز👇
http://t.me/joinchat/j13JylVQxgFiNjc8

**Comment by Webamoozcom** [comment]
Twitter (https://twitter.com/ReihaniHamed/status/1432237827111243781?s=19)
hamed reihani
#کلاهبردای جدید بنام #ارزدیجیتال #نوکون داره انجام میشه، متاسفانه دیشب به ۱۰۰۰ پیج #اینستاگرام پربازدید تبلیغ دادن

**Comment by mboveiri** [comment]
تبلیغ در کانال "تجارت نیوز" یکی از پرمخاطب ترین رسانه های اقتصادی ایران :

![image](https://user-images.githubusercontent.com/4364319/137625621-f18d464c-2794-4ba3-ba42-333ed01fa83c.png)

آرشیو : https://archive.ph/XJTdC

**Comment by M-R-FARHADI** [comment]
اسکم خالص الانم که توی  اکسچنجر کلاهبداری لیست کردن
